catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java

```diff
@@ -51,7 +58,28 @@ public HistorianPlugin(CatalogFramework catalogFramework) {
 
     @Override
     public CreateResponse process(CreateResponse input) throws PluginExecutionException {
-        return input;
+        List<Metacard> historyMetacards = input.getCreatedMetacards()
+                .stream()
+                .filter(m -> HistoryMetacardImpl.getVersionHistoryMetacardType()
+                        .getName()
+                        .equals(m.getMetacardType()
+                                .getName()))
+                .collect(Collectors.toList());
+
+        if (historyMetacards.isEmpty()) {
+            return input;
+        }
+
+        List<Metacard> resultMetacards = new ArrayList<>(input.getCreatedMetacards());
+        resultMetacards.removeAll(historyMetacards);
+
+        Map<String, Serializable> properties =
+                input.getProperties() != null ? input.getProperties() : new HashMap<>();
+        properties.put(HISTORY_METACARDS, new ArrayList<Serializable>(historyMetacards));
```

Comment:
@pklinef Updated update and delete to include the history metacards in properties as well, but will put up as a separate PR along with content change.